magda-web-client/src/config.js

```diff
@@ -4,7 +4,7 @@ import Format from "./Components/SearchFacets/Format";
 import Region from "./Components/SearchFacets/Region";
 import Temporal from "./Components/SearchFacets/Temporal";
 
-const fallbackApiHost = "//magda-dev.terria.io/";
+const fallbackApiHost = "https://magda-dev.terria.io/";
```

Comment:
I suspect this needs to stay // for local http development to work?

Reply:
Magda-dev should always work in HTTPS from now on I think... this is even better really seeing how https://magda-dev just gets redirected.

Reply:
👍